ch2.md

```diff
@@ -38,7 +38,7 @@ greeting('times');
 // "Hi times"
 ```
 
-In other words, `hi` is already a function that expects one argument, why place another function around it that simply calls `hi` with the same bloody argument? It doesn't make any damn sense. It's like donning your heaviest parka in the dead of July just to blast the air and demand an ice lolly.
+In other words, `hi` is already a function that expects one argument, why place another function around it that simply calls `hi` with the same bloody argument? It doesn't make any darned sense. It's like donning your heaviest parka in the dead of July just to blast the air and demand an ice lolly.
```

Comment:
I think we lose a bit of the original prose here. The sense is indeed preserved yet I would rather keep the original form.